windpowerlib/wind_farm.py

```diff
@@ -402,8 +403,10 @@ def assign_power_curve(self, wake_losses_model='wind_farm_efficiency',
                         wind_farm_power_curve['value'].values,
                         wind_farm_efficiency=self.efficiency))
             else:
-                logging.info("`wake_losses_model` is {} but wind farm ".format(
-                    wake_losses_model) + "efficiency is NOT taken into "
-                                         "account as it is None.")
+                msg = (
+                    "If you use `wake_losses_model` '{model}' your WindFarm "
+                    "needs an efficiency but `efficiency` of {farm} is {eff}.")
+                raise ValueError(msg.format(model=wake_losses_model, farm=self,
+                                            eff=self.efficiency))
```

Comment:
The output of `farm` is quite long and people may miss the rest of the message. Is it possible to change the order?

Something like this:
```diff
- "If you use `wake_losses_model` '{model}' your WindFarm "
- "needs an efficiency but `efficiency` of {farm} is {eff}.")
+ "If you use `wake_losses_model` '{model}' your WindFarm "
+ "needs an efficiency but `efficiency` is {eff}.\n\n"
+ "Failing farm:\n {farm}")

Reply:
Good point, thanks!